.github/workflows/playwright.yml.disabled

```diff
@@ -0,0 +1,39 @@
+# Does not work, because Adobe flags the source IP as untrusted, and requires confirming a code sent by email.
+
+name: Playwright Tests
```

Comment:
Is this workflow needed? I see this workflow will only run when we push any code to `main` branch.

Reply:
@shiv-gup 

I believe this entire file is disabled due to the name being changed to .yml.disabled. 

Definitely calls into question the usefulness of the entire file and the e2e file, but I don't believe this should cause an issue in the main branch.

Reply:
Discussed in slack. Agreed to leave as is for now as it is disabled and understand their purpose later.